desktop/install/linux-install.md

```diff
@@ -268,6 +270,17 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the default (and currently only) mechanism to enable file sharing between the host and Docker Desktop VM. In order not to require elevated privileges, without unnecessarily restricting operations on the shared files, Docker Desktop runs the file sharing service (`virtiofsd`) inside a [user namespace](https://man7.org/linux/man-pages/man7/user_namespaces.7.html) with UID and GID mapping configured. As a result Docker Desktop will rely on the host to be configured to enable the current user to use subordinate ID delegation. For this to be true [/etc/subuid](https://man7.org/linux/man-pages/man5/subuid.5.html) and [/etc/subgid](https://man7.org/linux/man-pages/man5/subgid.5.html) have to be present (Docker Desktop currently only supports subordinate ID delegation configured via files). In case of both Docker Desktop will use the first entry corresponding to the current user, to set up mappings for IDs above 0, and map the current users ID and GID to 0.
+
+In case `/etc/subuid` and `/etc/subgid` are missing they will need to be created. Both should contain entries in the following form `<username>:<start of id range>:<id range size>`. E.g. to allow a user whose username is `docker` to use IDs from 100000 to 165535:
+
+    sudo echo 'docker:100000:65536' > /etc/subuid
+    sudo echo 'docker:100000:65536' > /etc/subgid
```

Comment:
This may not work; the `sudo` applies to the `echo`, but not to the pipe; https://unix.stackexchange.com/questions/1416/redirecting-stdout-to-a-file-you-dont-have-write-permission-on

```bash
sudo echo 'bla' > /etc/bla
bash: /etc/bla: Permission denied

echo 'bla' | sudo tee /etc/bla
bla

cat /etc/bla
bla
```

We should also use a `console` typehint here, and add a prompt (`$`);

    ```console
    $ echo 'docker:100000:65536' | sudo tee /etc/subuid
    $ echo 'docker:100000:65536' | sudo tee /etc/subgid
    ```

Reply:
Thanks :) I have a tendency to forget that `sudo` does not work through pipes and redirections

Reply:
Same here! 😂 I recalled a similar thing in our install docs, and just the other day there was a similar issue, so I had it fresh in mind; https://github.com/docker/docker.github.io/pull/15243